wolfssl/wolfcrypt/asn.h

```diff
@@ -2194,6 +2194,11 @@ WOLFSSL_LOCAL int GetMyVersion(const byte* input, word32* inOutIdx,
 WOLFSSL_LOCAL int GetInt(mp_int* mpi, const byte* input, word32* inOutIdx,
                          word32 maxIdx);
 
+#ifndef WOLFSSL_ASN_TEMPLATE
+WOLFSSL_ASN_API int GetASNInt(const byte* input, word32* inOutIdx, int* len,
```

Comment:
Let's add a version for WOLFSSL_ASN_TEMPLATE. I think its similar to template's GetInt. @SparkiDev should help with this.

```
ASNGetData dataASN[intASN_Length];
XMEMSET(dataASN, 0, sizeof(dataASN));
ret = GetASN_Items(intASN, dataASN, intASN_Length, 0, input, inOutIdx, maxIdx);
```

Reply:
I took @SparkiDev 's suggestion and just exposed `GetASNInt()` even when `ASN_TEMPLATE` is enabled.